packages/core/admin/server/controllers/api-token.js

```diff
@@ -63,4 +67,42 @@ module.exports = {
 
     ctx.send({ data: apiToken });
   },
+
+  async update(ctx) {
+    const { body } = ctx.request;
+    const { id } = ctx.params;
+    const apiTokenService = getService('api-token');
+
+    /**
+     * We trim both field to avoid having issues with either:
+     * - having a space at the end or start of the value.
+     * - having only spaces as value;
+     */
+    const attributes = {
+      name: trim(body.name),
+      description: trim(body.description),
+      type: body.type,
+    };
```

Comment:
Since description is optional, setting it here will result in updating it anyway to `''`

Reply:
The default value for the description is `''` so this is not a problem. Unless we allow for partial payload. Then, yes, I'll having to wrap the each property in an `if` statement to only apply the `trim` if it exists.

Reply:
Alright, I understand your point of view. `description` being optional confused me, not knowing if it's a partial update or full update :p